src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -28,6 +30,8 @@ public class ClusterTransactionCommand : CommandBase
         public string ClusterTransactionId;
 
         public long DatabaseCommandsCount;
+        //We take the current ticks in advance to ensure consist results of the command execution on all nodes
+        public long CommandCreationTicks;
```

Comment:
What would happened in a mixed cluster? this property would be `0` when sent from an old node.

Reply:
In this case, the current ticks are the lowest value
so it will not be after (grater) than the `Expires` ticks.
I will set it to `long.MinValue` to be more explicit